src/scripts/main.js

```diff
@@ -1,28 +1,47 @@
 'use strict';
 
 const food = {
-  'Drink': {
-    'Wine': {},
-    'Schnaps': {},
+  Drink: {
+    Wine: {},
+    Schnaps: {},
   },
 
-  'Fruit': {
-    'Red': {
-      'Cherry': {},
-      'Strawberry': {},
+  Fruit: {
+    Red: {
+      Cherry: {},
+      Strawberry: {},
     },
-    'Yellow': {
-      'Banana': {},
-      'Pineapple': {},
+    Yellow: {
+      Banana: {},
+      Pineapple: {},
     },
-
   },
 };
 
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
   // WRITE YOUR CODE HERE
+  if (element.id === 'tree') {
+    const ul = document.createElement('ul');
+
+    element.append(ul);
+    createTree(ul, data);
+  } else {
+    for (const key of Object.keys(data)) {
+      const li = document.createElement('li');
+
+      li.textContent = key;
+      element.appendChild(li);
+
+      if (Object.keys(data[key]).length !== 0) {
+        const ul = document.createElement('ul');
+
+        li.appendChild(ul);
```

Comment:
after the previous comment fix you can remove it

Reply:
If I make changes in the code as you suggest, I get incorrect ul structure
![Screenshot 2023-01-22 at 20 00 54](https://user-images.githubusercontent.com/22715618/213932206-6793b6ab-9344-4301-a9b3-7a338a3dc148.png)
![Screenshot 2023-01-22 at 20 01 05](https://user-images.githubusercontent.com/22715618/213932207-93058a84-7576-42bb-a9fe-3c66e5baabbf.png)
![Screenshot 2023-01-22 at 20 01 12](https://user-images.githubusercontent.com/22715618/213932208-6bcfbfe3-8d09-443b-81a4-1904595f32d5.png)